src/repositories/schemas.ts

```diff
@@ -1,9 +1,195 @@
 import { OAuth1Client, ObjectId } from '@extrahorizon/javascript-sdk';
 
+interface Transition {
+  name: string;
+  type: string;
+  fromStatuses: string[];
+  toStatus: string;
+  conditions: { type: string; };
+  actions: { type: string; };
+  afterActions: { type: string; };
+}
+
 export async function remove(sdk: OAuth1Client, schemaId: ObjectId) {
```

Comment:
I know you didn't do this, but you made another comment about not wanting to update all of the contracts in this file with in this PR, but I definitely think we should make a ticket to do so.

we have things like `createSchema,` but then the `deleteSchema` is just `remove`

I wonder if these interfaces should be schema.properties.create(''), as it's all the same object rather than createProperty, just a thought and definitely not for this PR.

Reply:
Will make a ticket to improve this file, out of scope for this PR but definitely needs to be done. Ticket will include looking at the naming and change to raw calls to actual calls.

Reply:
https://extrahorizon.atlassian.net/browse/SVCS-2810